src/App/component/Sidebar.jsx

```diff
@@ -1,500 +1,519 @@
-    </div>
-  )
-}
-
-const mapStateToProps = (state) => {
-  return {
-    startMonth: state.selectedMonth.startMonth,
-    endMonth: state.selectedMonth.endMonth,
-    currentProjectId: state.currentProjectId
-  }
-}
-
-const mapActionToProps = (dispatch) => {
-  return {
-    setStartMonth: (startMonth) => dispatch(setStartMonth(startMonth)),
-    setEndMonth: (endMonth) => dispatch(setEndMonth(endMonth))
-  }
-}
-
-export default connect(mapStateToProps, mapActionToProps)(Sidebar)
+import {useEffect, useState} from 'react'
+import logo_p from '../../assets/p.png'
+import logo_v from '../../assets/v.png'
+import logo_s from '../../assets/s.png'
+import {useHistory} from 'react-router-dom'
+import {
+  ExitToApp,
+  ArrowBack,
+  ExpandLess,
+  ExpandMore,
+  Code,
+  GpsFixed,
+  Compare
+} from '@material-ui/icons'
+import {
+  Drawer,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  CssBaseline,
+  AppBar,
+  Toolbar,
+  Divider,
+  Collapse,
+  IconButton
+} from '@material-ui/core'
+import {AiFillBug} from 'react-icons/ai'
+import {IoGitCommitSharp, IoNuclear} from 'react-icons/io5'
+import {GoIssueOpened} from 'react-icons/go'
+import {HiDocumentDuplicate} from 'react-icons/hi'
+import {SiGithub, SiSonarqube, SiGitlab, SiTrello} from 'react-icons/si'
+import {RiDashboardFill} from 'react-icons/ri'
+import clsx from 'clsx'
+import {MuiPickersUtilsProvider, DatePicker} from '@material-ui/pickers'
+import MomentUtils from '@date-io/moment'
+import {makeStyles} from '@material-ui/core/styles'
+import {connect} from 'react-redux'
+import {setStartMonth, setEndMonth} from '../../redux/action'
+import Axios from 'axios'
+
+const drawerWidth = 240
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+  },
+  appBar: {
+    zIndex: theme.zIndex.drawer + 1,
+    transition: theme.transitions.create(['width', 'margin'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+  },
+  menuButton: {
+    marginRight: 36,
+  },
+  hide: {
+    display: 'none',
+  },
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+  },
+  drawerOpen: {
+    width: drawerWidth,
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  },
+  drawerClose: {
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    overflowX: 'hidden',
+    width: theme.spacing(7) + 1,
+    [theme.breakpoints.up('sm')]: {
+      width: theme.spacing(9) + 1,
+    },
+  },
+  drawerContent: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    padding: theme.spacing(0, 1),
+    ...theme.mixins.toolbar,
+  },
+  content: {
+    flexGrow: 1,
+    padding: theme.spacing(3),
+  },
+  // search: {
```

Comment:
Suggest remove commented code, please see this [article](https://kentcdodds.com/blog/please-dont-commit-commented-out-code) for more details.